src/frontend/components/Chat/SharedChatComponents/ChatMessageItem/index.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react';
+import { DateTime } from 'luxon';
+import React from 'react';
+import { ChatMessageItem } from '.';
+import { wrapInIntlProvider } from '../../../../utils/tests/intl';
+
+describe('<ChatMessageItem />', () => {
+  it('renders the chatMessagesMetadatas and checks displayed informations and sizes are corrects.', () => {
```

Comment:
```suggestion
  it('renders the chatMessagesMetadatas and checks displayed information and sizes are correct.', () => {
```

Reply:
done in [e3aa8c3dd2a1891a4111593234da3f25e1c08a76](https://github.com/openfun/marsha/pull/1247/commits/e3aa8c3dd2a1891a4111593234da3f25e1c08a76)